src/connectors/components/HeaderMenuRequestConnector.js

```diff
@@ -37,7 +37,7 @@ export const mapStateToProps = (state: State) => {
   const dropDownButtons = uniqueLegacyAddress ? [addressToggle, help] : [help]
   return {
     style: dropDownStyle,
-    dropDownButtons,
+    data: dropDownButtons,
```

Comment:
This is passing a statically allocated Object as a prop. This will ALWAYS cause invalidation of props and a re-render even if nothing changed. This is especially costly in performance because it's in the header which is always mounted.

Reply:
So, here is the issue, and I am open to suggestions on a better way to do this, 
The requirement is the drop down menu needs to have two buttons sometimes, other times it is one button. When the second button is visible, its content needs to change. on whether or not this button displays the content to select regular address or legacy address, so unfortunately it is set up to behave the way it needs to.  If there is a better way to do this, I am open, however this is designed to check itself and re-render.